cmd/tikv-ctl/src/cmd.rs

```diff
@@ -548,7 +548,11 @@ pub enum Cmd {
         version: u64,
     },
     /// Control for Raft Engine
-    RaftEngineCtl { args: Vec<String> },
+    /// Usage: tikv-ctl raft-engine-ctl -- --help
+    RaftEngineCtl {
```

Comment:
do you update the doc?

Reply:
You mean pingcap/docs? I haven't. raft-engine-ctl is a bit complicated in itself, I may publish it to docs after I (or someone else) creates a dedicated troubleshooting.md for it.